internal/ui/login/static/templates/footer.html

```diff
@@ -1,3 +1,7 @@
 {{define "footer"}}
-
+<footer>
+    <span class="fill-space"></span>
+    <a href="{{t "Footer.TosLink"}}" target="_blank" alt="Privacy Policy">{{t "Footer.TosPrivacyPolicy"}}</a>
```

Comment:
do we really have / want to place the TOS into the footer? aren't they only relevant for global org users and org admins? @fforootd

Reply:
I would recommend that we provide a way to see the TOS and Privacy Policy